src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -389,6 +389,35 @@ g.test('sampleCount,valid_sampleCount_with_other_parameter_varies')
     }, !success);
   });
 
+g.test('sample_count,1d_2d_array_3d')
+  .desc(
+    `
+Test that you can not create 1d, 2d_array, and 3d multisampled textures
+`
+  )
+  .params(u =>
+    u.combineWithParams([
+      { dimension: '1d', size: [4, 1, 1] },
+      { dimension: '2d', size: [4, 4, 4] },
+      { dimension: '2d', size: [4, 4, 6] }, // possibly cube
+      { dimension: '3d', size: [4, 4, 4] },
+    ] as const)
+  )
+  .fn(async t => {
+    const { dimension, size } = t.params;
+
+    t.expectValidationError(() => {
+      const texture = t.device.createTexture({
+        size,
+        dimension,
+        sampleCount: 4,
+        format: 'rgba8unorm',
+        usage: GPUTextureUsage.TEXTURE_BINDING | GPUTextureUsage.RENDER_ATTACHMENT,
+      });
+      assert(!!texture, 'created texture');
```

Comment:
nit: Theoretically not necessary because createTexture can't return anything other than GPUTexture, though not hurting anything

Reply:
removed. I only added it because of the complaint that `texture` is never used